modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-installation.adoc

```diff
@@ -0,0 +1,98 @@
+[[installation]]
+== Installation
+
+[NOTE]
+A valid license key is needed in order to install {pluginname} On-Premises.
+link:https://www.tiny.cloud/contact/[Contact us] for a trial license key.
+
+=== Supported technologies
+
+The application is provided as a docker image by default.
+
+It can be run with any Open Container runtime tool e.g. link:https://kubernetes.io/[Kubernetes], link:https://www.redhat.com/en/technologies/cloud-computing/openshift[OpenShift], link:https://podman.io/[Podman], link:https://docs.docker.com/[Docker] and many others.
+
+Refer to the xref:individual-export-to-pdf-on-premises.adoc#requirements[Requirements guide] for more information about the hardware and software requirements to run the {pluginname} On-Premises.
+
+=== Setting up the application using a Docker container
+
+. The username and password credentials supplied by Tiny are utilized for logging into the Docker registry and retrieving the Docker image.
+. Containerize the application using `docker` or `docker-compose`.
+. Use a demo page to verify if the application works properly.
+
+==== Containerize example using docker
+
+Login to Docker registry:
+
+[source, sh, subs="attributes+"]
+----
+docker login -u [username] -p [password] registry.containers.tiny.cloud
+----
+
+Launch the Docker container:
+
+[source, sh, subs="attributes+"]
+----
+docker run --init -p 8080:8080 -e LICENSE_KEY=[your_license_key] {dockerimageexporttopdf}:[version]
+----
+
+If using authorization provide the SECRET_KEY:
+
+[source, sh, subs="attributes+"]
+----
+docker run --init -p 8080:8080 -e LICENSE_KEY=[your_license_key] -e SECRET_KEY=[your_secret_key] {dockerimageexporttopdf}:[version]
+----
+
+Read more about using authorization in the xref:individual-export-to-pdf-on-premises.adoc#authorization[authorization] section.
+
+==== Containerize example using docker-compose
+
+. Create the docker-compose.yml file:
++
+[source, yml, subs="attributes+"]
+----
+version: "3.8"
+services:
+  pdf-converter:
```

Comment:
@ShiridiGandham should this be `pdf-converter-tiny` to match?
```suggestion
  pdf-converter-tiny:
```

Reply:
Confirmed.